src/model/dfcxx/include/dfcxx/io.h

```diff
@@ -28,6 +28,10 @@ class IO {
 
   DFVariable inputScalar(const std::string &name, const DFType &type);
 
+  DFVariable hollow(const DFType &type);

```

Comment:
What is "hollow" synonym here? What does this method do exactly?

Reply:
hollow -> unconnected?
hollow -> dummy?

It would be good to have more describing name for the method here

Reply:
I get it. `Unconnected` - too long. `Dummy` - too confusing. I am going to check MaxCompiler docs on how they called these things.